frontend/app/src/utils/constants.js

```diff
@@ -76,6 +76,10 @@ export const RunTypes = Object.freeze({
 
 /** Status Codes */
 export const StatusCode = Object.freeze({
```

Comment:
let's rename all the artifacts to row status code instead of status code or better check with business should it be plr record status code?

Reply:
changed the const name to RowStatusCode, And there are some codes that are PHLAT tool-specific as well.